scripts/coverage.sh

```diff
@@ -2,4 +2,4 @@
 
 rm -rf flat
 
-TRAVIS_PULL_REQUEST=true scripts/test.sh
\ No newline at end of file
+TRAVIS_PULL_REQUEST=false scripts/test.sh
```

Comment:
This will disable coverage calculation on local machines but won't affect travis as travis sets `TRAVIS_PULL_REQUEST` to the pull request number automatically (which apparently, can't be overwritten).